src/app/common/paging/quick-column-toggle/quick-column-toggle.component.ts

```diff
@@ -29,24 +29,25 @@ export class QuickColumnToggleComponent extends QuickFiltersComponent {
 	private checkColumnConfiguration() {
 		// Check first to see if all columns are turned on
 		this.columnMode = 'all';
-		this.filterKeys.some((key: string) => {
```

Comment:
What was the issue that sonarqube flagged here?  The update changes the logical flow.  The `return true;` below was returning out of the inline function executed by the iterator.   With this change it is now returning out of the `checkColumnConfiguration` function.  This has the negative side effect that it will skip the logic below that checks if the default columns are enabled.

Reply:
Was it due to the `some` call having side effects (setting `columnMode`)?  The logic flow issue could be fixed by changing the return to a break, but here is a better more concise solution.

```ts
private checkColumnConfiguration() {
	if (this.filterKeys.every(key => this.filters[key].show)) {
		this.columnMode = 'all';
	} else if (
		this.filterKeys.some(key => this.filters[key].show !== this.defaultFilters[key].show)
	) {
		this.columnMode = 'custom';
	} else {
		this.columnMode = 'default';
	}
}
```

Reply:
This got flagged as a minor issue because the return value of the `some` call was being ignored. Good catch on the logical flow change! I've replaced the function with the new one you provided.